packages/core/src/connectivity/scp-cf/resilience-options.ts

```diff
@@ -8,5 +8,6 @@ export interface ResilienceOptions {
 export const circuitBreakerDefaultOptions = {
   timeout: 10000,
   errorThresholdPercentage: 50,
+  volumeThreshold: 10,
```

Comment:
We now have more hard coded options, should we make a follow up so users can customize the options?

Reply:
I think users should not be able to customize the options on which we break for the destination and xsuaa service as this is meant to protect the BTP services. At least that is how we designed it initially and how I discussed it with @FrankEssenberger recently. That said, the timeout will be removed from those options and will be made configurable.